packages/main/src/plugin/updater.ts

```diff
@@ -166,6 +169,56 @@ export class Updater {
    */
   private onUpdateAvailable(): void {
     this.updateAvailableEntry();
+    if (this.getConfigurationValue() === 'startup') {
+      this.commandRegistry.executeCommand('update').catch((err: unknown) => {
+        console.error('Something went wrong while executing update command', err);
+      });
+    }
+  }
+
+  /**
+   * Registers configuration settings for update preferences.
+   */
+  private registerConfiguration(): void {
+    this.configurationRegistry.registerConfigurations([
+      {
+        id: 'preferences.update',
+        title: 'Update',
+        type: 'object',
+        properties: {
+          ['preferences.update.reminder']: {
+            description: 'Update reminder',
```

Comment:
I think we need to be more verbose in the description
just a suggestion, can be any other wording
`Configure whether you receive update reminders when starting Podman Desktop`